src/common/Collision/Management/IVMapManager.h

```diff
@@ -55,7 +55,8 @@ namespace VMAP
 
             virtual int loadMap(const char* pBasePath, unsigned int pMapId, int x, int y) = 0;
 
-            virtual bool existsMap(const char* pBasePath, unsigned int pMapId, int x, int y) = 0;
+            ///Changed from Bool to Int, in order to be able to use the same method with a little bit more of error descriptor.
```

Comment:
Remove this comment, it doesn't really add anything.

Change the int to a new enum describing the possible return states:


Something like this would do:

```c++
enum class MapLoadError
{
     None,
     FileNotFound,
     VersionMismatch,
     Other
}